### PR TITLE
Create template react app

### PR DESCRIPTION
**Pull request by JiriLojda**
### Motivation

Which issue does this fix? Fixes #`issue number`

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [x] Code follows coding conventions held in this repo
- [x] Automated tests have been added
- [x] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,67 +7,23 @@
...
-<!-- LICENSE -->
 ## License
 
 Distributed under the MIT License. See [`LICENSE.md`](./LICENSE.md) for more information.
 
 
-
-<!-- MARKDOWN LINKS & IMAGES -->
-<!-- https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#badges-->
 [contributors-shield]: https://img.shields.io/github/contributors/Kentico/kontent-custom-element-samples.svg?style=for-the-badge
```
Comment (IvanKiral) [inline README.md]: There are wrong links for badges

+ nitpick: markdown lint is showing me some warnings like trailing whitespaces :D

**Review by IvanKiral — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JiriLojda) [reply README.md]: I updated the links but I can't see any markdown lint suggestions in my IDE.

File: README.md [hunk 2]
```diff
@@ -7,76 +7,32 @@
...
+[contributors-url]: https://github.com/kontent-ai/integration-template-react/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/kontent-ai/integration-template-react.svg?style=for-the-badge
+[forks-url]: https://github.com/kontent-ai/integration-template-react/network/members
+[stars-shield]: https://img.shields.io/github/stars/kontent-ai/integration-template-react.svg?style=for-the-badge
+[stars-url]: https://github.com/kontent-ai/integration-template-react/stargazers
+[issues-shield]: https://img.shields.io/github/issues/kontent-ai/integration-template-react.svg?style=for-the-badge
+[issues-url]:https://github.com/kontent-ai/integration-template-react/issues
+[license-shield]: https://img.shields.io/github/license/kontent-ai/integration-template-react.svg?style=for-the-badge
+[license-url]:https://github.com/kontent-ai/integration-template-react/blob/master/LICENSE.md
 [discussion-shield]: https://img.shields.io/discord/821885171984891914?color=%237289DA&label=Kontent%20Discord&logo=discord&style=for-the-badge
```
Comment (IvanKiral) [inline README.md]: Would it be possible to have kontent.ai badge?

**Comment by IvanKiral** [comment]
Internally we agreed to rework the config to contain some property to show the use case

**Review by IvanKiral — CHANGES_REQUESTED** [review]

**Review by strizr — APPROVED** [review]

**Review by IvanKiral — APPROVED** [review]

**Pull request merged**